Labs/AI and Machine Learning/Video Indexer/Resources/index.js

```diff
@@ -7,8 +7,7 @@ let win
 function createWindow() {
    win = new BrowserWindow({width: 1000, height: 700})
 
-   win.setMenu(null); 
-   win.setIcon(path.join(__dirname, 'assets/icons/ico/app.ico'));    
```

Comment:
This function is not supported on macOS, so removing to ensure it works cross-platform